entr.1

```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
 waits for the
 .Ar utility
 to exit to ensure that resources such as sockets have been closed.
-Control of the TTY is not transferred the child process.
+Control of the TTY is not transferred to the child process.
 .It Fl s
-Evaluate the first argument using the interpreter specified by the
-.Ev SHELL
-environment variable.
-If stdout is a TTY, the name of the shell and exit code is printed after each
-invocation.
+Evaluate the first argument using the interpreter specified by
+.Ev SHELL .
+If standard output is a TTY, the name of the interpreter and exit code is
+printed after each invocation.
 .El
 .Pp
 The first argument named
-.Ar /_
+.Dq Ar /_
 is replaced with the absolute path of the first file to trigger an event.
 The first file under watch is used as the default.
 If the
 .Fl s
-option is used, the name of the first file to triggler an event can be read from
+option is used, the name of the first file to trigger an event can be read from
 .Va $0 .
-.Sh COMMANDS
+.Ss Commands
 .Nm
 listens for keyboard input and responds to the following commands:
-.Bl -tag -width Ic
-.It SPACE
-Execute the utility immediately.
+.Bl -tag -width <space>
+.It Aq Cm space
+Execute
+.Ar utility
+immediately.
 If the
 .Fl r
 option is set this will terminate and restart the child process as if a file
 change event had occurred.
-.It q
-Quit; equivalent pressing Ctrl\-C.
+.It Cm q
+Quit
+.Nm .
+Equivalent to pressing
+.Aq Cm control-c
+or sending
+.Dv SIGINT .
 .El
 .Sh ENVIRONMENT
-If
-.Ev PAGER
-is undefined,
+.Bl -tag -width PAGER
+.It Ev PAGER
+If undefined,
 .Nm
-will assign
-.Pa /bin/cat
-to prevent interactive utilities from waiting for
+assigns it to
+.Xr cat 1 .
+This prevents interactive utilities from waiting for
 keyboard input if output does not fit on the screen.
-.Pp
-If
-.Ev SHELL
-is undefined,
-.Nm entr
-will use
-.Pa /bin/sh .
+.It Ev PAGER
+Execute
+.Ar utility
+with the current shell.
+If undefined,
+.Nm
+assigns it to
+.Xr sh 1 .
+.El
 .Sh EXIT STATUS
 The
 .Nm
 utility exits with one of the following values:
 .Pp
-.Bl -tag -width Ds -offset indent -compact
-.It Li 0
```

Comment:
ok suggestion, but I'm going to keep it the same as https://man.openbsd.org/diff